### PR TITLE
change repoistory and account

### PR DESCRIPTION
**Pull request by bellatoris**
## Checklist
- [ ] 충분한 양의 자동화 테스트를 작성했는가?
  - 계단정복지도 서비스는 사이드 프로젝트로 진행되는 만큼 충분한 QA 없이 배포되는 경우가 많습니다. 따라서 자동화 테스트를 꼼꼼하게 작성하는 것이 서비스 품질을 유지하는 데 매우 중요합니다.

### CODE REVIEW

**Comment by bellatoris** [comment]
Current dependencies on/for this PR:
* main
  * **PR #172** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/172" target="_blank">Graphite</a> 
    * **PR #173** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/173" target="_blank">Graphite</a> 
      * **PR #174** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/174" target="_blank">Graphite</a> 
        * **PR #175** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/175" target="_blank">Graphite</a> 
          * **PR #176** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/176" target="_blank">Graphite</a> 
            * **PR #177** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/177" target="_blank">Graphite</a> 
              * **PR #181** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/181" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/181?utm_source=stack-comment).

**Review by Zeniuus — APPROVED** [review]

File: app-admin-frontend/docker-push.sh [hunk 1]
```diff
@@ -14,5 +14,5 @@ aws ecr-public get-login-password --region us-east-1 | docker login --username A
 
 npm install
 npm run "build:$1"
-docker buildx build --platform linux/amd64 -t public.ecr.aws/q0g6g7m8/scc-admin-frontend:$IMAGE_TAG .
-docker push public.ecr.aws/q0g6g7m8/scc-admin-frontend:$IMAGE_TAG
+docker buildx build --platform linux/amd64 -t public.ecr.aws/i6n1n6v2/scc-admin-frontend:$IMAGE_TAG .
```
Comment (Zeniuus) [inline app-admin-frontend/docker-push.sh:17]: 이거 위에 ecr login 하는 곳이 안 바뀌어 있어유

텍스트로 전체검색 해야 할 듯?

**Review by Zeniuus — CHANGES_REQUESTED** [review]

File: app-admin-frontend/docker-push.sh (hunk 1 above)
Comment (bellatoris) [reply app-admin-frontend/docker-push.sh:17]: no way

Comment (Zeniuus) [reply app-admin-frontend/docker-push.sh:17]: [73725af](https://github.com/Stair-Crusher-Club/scc-server/pull/183/commits/73725afe531bef172267b83d9972871ab0cf9f51) 걍 #183 에서 처리함

**Review by Zeniuus — APPROVED** [review]

**Comment by bellatoris** [comment]
@bellatoris started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/181).

File: app-admin-frontend/docker-push.sh (hunk 1 above)
Comment (bellatoris) [reply app-admin-frontend/docker-push.sh:17]: 감사합니다

**Comment by bellatoris** [comment]
[Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/181) failed to merge this pull request for an unknown reason. Try your merge again, or report a bug if you see this consistently.

**Comment by bellatoris** [comment]
@bellatoris started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/181).

**Comment by bellatoris** [comment]
@bellatoris merged this pull request with [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/181).

**Pull request merged**